.github/actions/prepare-node-deps/action.yaml

```
@@ -21,6 +21,14 @@ runs:
         node node_modules/\@sentry/cli/scripts/install.js
         echo "::endgroup::"
 
+    - name: Cache Node.js modules
+      uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
+      with:
+        path: ~/.npm
+        key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.OS }}-node-
```

```suggestion
        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
        restore-keys: |
          ${{ runner.os }}-node-
```
afaik Github is not case independent, when it comes to context access